docs/schedulers.md

```diff
@@ -41,43 +42,43 @@ from syne_tune.optimizer.schedulers.fifo import FIFOScheduler
 from syne_tune.tuner import Tuner
 from syne_tune.stopping_criterion import StoppingCriterion
 
-from examples.training_scripts.mlp_on_fashion_mnist.mlp_on_fashion_mnist import \
-    mlp_fashionmnist_benchmark
+from examples.training_scripts.mlp_on_fashion_mnist.mlp_on_fashion_mnist import
 
+mlp_fashionmnist_benchmark
 
 if __name__ == '__main__':
-    logging.getLogger().setLevel(logging.DEBUG)
-    n_workers = 4
-
-    # We pick the MLP on FashionMNIST benchmark
-    # The 'benchmark' dict contains arguments needed by scheduler and
-    # searcher (e.g., 'mode', 'metric'), along with suggested default values
-    # for other arguments (which you are free to override)
-    benchmark = mlp_fashionmnist_benchmark({'dataset_path': './'})
-    config_space = benchmark['config_space']
-
-    backend = LocalBackend(entry_point=benchmark['script'])
-
-    # GP-based Bayesian optimization searcher. Many options can be specified
-    # via `search_options`, but let's use the defaults
-    searcher = 'bayesopt'
-    search_options = {'num_init_random': n_workers + 2}
-    # FIFOScheduler. Together with searcher `bayesopt`, this selects Bayesian
-    # optimization without early stopping.
-    scheduler = FIFOScheduler(
-        config_space,
-        searcher=searcher,
-        search_options=search_options,
-        mode=benchmark['mode'],
-        metric=benchmark['metric'])
-
-    tuner = Tuner(
-        backend=backend,
-        scheduler=scheduler,
-        stop_criterion=StoppingCriterion(max_wallclock_time=120),
-        n_workers=n_workers)
-
-    tuner.run()
+  logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
is there a reason why you change the indent?

Reply:
space versus tab in IDE I guess, we should use black to avoid this.